tests/e2e/ctl_v3_snapshot_test.go

```diff
@@ -29,8 +29,7 @@ import (
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
-func TestCtlV3Snapshot(t *testing.T)        { testCtl(t, snapshotTest) }
```

Comment:
I don't think we should remove a test in this PR. Why remove this one?

Reply:
> I don't think we should remove a test in this PR. Why remove this one?

The test `TestCtlV3Snapshot` is same as `TestCtlV3SnapshotEtcdutl` when delete  `withEtcdutl`

Reply:
The test `TestCtlV3Snapshot` is same as `TestCtlV3SnapshotEtcdutl` when delete `withEtcdutl`.

Reply:
Is `TestCtlV3Snapshot` is currently using etcutl?

Reply:
It‘s used both. 
Command `snapshot save` would use etcdctl like https://github.com/kkkkun/etcd/blob/62641d33854e206f915398332dbc2b976e9ae5c8/tests/e2e/ctl_v3_snapshot_test.go#L131,

and `snapshot restore/status` would use etcdutl. https://github.com/kkkkun/etcd/blob/62641d33854e206f915398332dbc2b976e9ae5c8/tests/e2e/ctl_v3_snapshot_test.go#L136

Reply:
So they don't test the same thing. One uses `etcdctl` other `etcutl`, based on that I don't think we want to remove the test.

Reply:
They test the same thing.  Both `TestCtlV3Snapshot` and `TestCtlV3SnapshotEtcdutl` call `snapshotTest`.

In `snapshotTest`, it firstly do `etcdctl snapshot save` used etcdctl , and then  `etcdutl snapshot status` used etcdutl.

Reply:
Ack, thanks for confirming.